packages/components/src/VirtualizedList/Content.component.js

```diff
@@ -5,23 +5,29 @@ import TooltipTrigger from '../TooltipTrigger';
 
 function DefaultRenderer({ cellData, columnData, rowData }) {
 	const { getTooltipLabel } = columnData;
-	let tooltipLabel = columnData.tooltipLabel || cellData;
+	let tooltipLabel = columnData.tooltipLabel != null ? columnData.tooltipLabel : cellData;
 	if (typeof getTooltipLabel === 'function') {
 		tooltipLabel = getTooltipLabel(rowData);
 	}
-	return tooltipLabel ? (
+	return tooltipLabel != null ? (
```

Comment:
![image](https://github.com/Talend/ui/assets/19857479/c8d29a96-2d3f-409d-bbc3-0360a1549be6)

Reply:
Yes that's expected , i wanted to exclude null and undefined so i used `!=` and not `!==`